tests/model_fields_/models.py

```diff
@@ -167,6 +167,11 @@ def __str__(self):
         return self.title
 
 
+class Audit(models.Model):
```

Comment:
This could include a comment as with the other museum models, and I'd order it below them all.